plugins/kubernetes/app/models/kubernetes/client_config_file.rb

```diff
@@ -21,6 +21,10 @@ def client_for(context_name)
       contexts[context_name].try(:client)
     end
 
+    def extension_client_for_client_for(context_name)
```

Comment:
Ugh, thanks for catching that… that was me fat fingering a function rename. It is being used! :/
> On 5 Jan 2016, at 11:18, Maciek Sufa <notifications@github.com> wrote:
> 
> In plugins/kubernetes/app/models/kubernetes/client_config_file.rb <https://github.com/zendesk/samson/pull/689#discussion_r48835104>:
> 
> > @@ -21,6 +21,10 @@ def client_for(context_name)
> >        contexts[context_name].try(:client)
> >      end
> >  
> > +    def extension_client_for_client_for(context_name)
> Kudos on the method name :) Since it's not used anywhere I presume this is some rubbish code?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/zendesk/samson/pull/689/files#r48835104>.
>